test/e2e_test.go

```diff
@@ -157,6 +165,24 @@ var verifyLocal = func(keyRef, path string, checkClaims bool, annotations map[st
 	return cmd.Exec(context.Background(), args)
 }
 
+var verifyOffline = func(keyRef, imageRef string, checkClaims bool, annotations map[string]interface{}, attachment string) error {
+	cmd := cliverify.VerifyCommand{
+		KeyRef:        keyRef,
+		RekorURL:      "notreal",
+		Offline:       true,
+		CheckClaims:   checkClaims,
+		Annotations:   sigs.AnnotationsMap{Annotations: annotations},
+		Attachment:    attachment,
+		HashAlgorithm: crypto.SHA256,
+		IgnoreTlog:    true,
```

Comment:
Can this be removed and the tests still pass? Otherwise we skip rekor verification

Reply:
This is supposed to be offline only, do we want online rekor verification?

Reply:
The Rekor proof _should_ be uploaded to the OCI annotation, and then read in during verification and not require contacting the log. Setting "offline" to "true" and providing the fake rekor URL will force offline verification.

For what it's worth, some of these tests are a bit old so there may be cases where this was turned off incorrectly. I would try to set ignoreTlog to false and see if all tests pass. The only case where it might not is if the test set TlogUpload to false (https://github.com/sigstore/cosign/blob/main/cmd/cosign/cli/options/sign.go#L36).

Reply:
> The Rekor proof should be uploaded to the OCI annotation

It's only uploaded if I set `TlogUpload` to true in the signing command, so I'll have to update all of those.

> I would try to set ignoreTlog to false and see if all tests pass

This is a net-new verify helper function. Should I turn off `IgnoreTlog` for all the other verify helpers as well?

Reply:
Ah yes, good point about having to update everything. I was thinking that TlogUpload is true by default, but that's only for the flag, not the CLI struct. Let's get this merged in then as-is and then we can circle back to improving e2e test coverage for Rekor verification later. Does that sound good?

Reply:
Sounds good :+1: